### DISCUSSION TITLE
[bug] pressTheme cursor pointer

### DISCUSSION THREAD

**Discussion by lobor** [tamagui/tamagui#554] [Ideas]
On Stack component, when use pressTheme props, i need set also cursor="pointer"

May be on default pressTheme add cursor on pointer

**Comment by dohomi** [comment]
I came across this issue as well, I also think that `pressTheme` should be add the css right away

**Comment by natew** [comment]
Coming in 98af183e750a6e40afcdd83a1b9897ab05a2f7c5
